iOS/DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -350,17 +364,25 @@ final class DefaultSubscriptionPagesUseSubscriptionFeature: SubscriptionPagesUse
 
         setTransactionStatus(.polling)
 
+        var subscriptionParameters: [String: String]?
+
         // Free Trials Experiment Parameters & Pixels
-        var freeTrialParameters: [String: String]?
         if shouldPerformFreeTrialPostPurchaseActions {
-            freeTrialParameters = completeSubscriptionFreeTrialParameters
+            subscriptionParameters = completeSubscriptionFreeTrialParameters
             fireFreeTrialSubscriptionPurchasePixel(for: subscriptionSelection.id)
         }
 
+        if let feExperiment = subscriptionSelection.experiment {
+            subscriptionParameters = {
+                let parameters = subscriptionParameters ?? [String: String]()
+                return parameters.merging(feExperiment.asParameters()) { $1 }
+            }()
+        }
```

Comment:
If we have an experiment in the subscription selection we pass the name and cohort as additional parameters.